### DISCUSSION TITLE
When are you releasing the setConsent commit made recently?

### DISCUSSION THREAD

**Discussion by sisundin** [invertase/react-native-firebase#7635] [Q&A] [answered]
Hi! The team and I are in the process implementing user set consent in our app. I see that you have made a [commit](https://github.com/invertase/react-native-firebase/commit/78169854f16a2715f5d2657ab08f54d5a4b05281) addressing this functionality yesterday and I'm just wondering about the release schedule is for this change. 

Keep up the good work, really appreciate this library!🥇

**Comment by mikehardy** [comment, marked as answer]
Hopefully later today, or early tomorrow.

In the meantime if you are handy with patch-package I implemented an automated patchset generator for all PRs and main merges such that people can get / test / use changes as soon as they want. Since we are a monorepo this would be kind of difficult otherwise, but with these patchsets you can get things pretty easily pre-publish now

They are uploaded as github artifacts on the related action summary page https://github.com/invertase/react-native-firebase/actions/runs/7963378973

**Reply by sisundin** [reply]
Awesome! Thank you for the quick response! Well that is great.

**Reply by mikehardy** [reply]
releasing now, should pop out as 18.9.0

**Reply by sisundin** [reply]
Works great! Thank you! :)
